ng-dev/pr/common/fetch-pull-request.ts

```diff
@@ -87,7 +89,12 @@ export const PR_SCHEMA = {
     {
       nodes: [
         {
+          login: graphqlTypes.string,
```

Comment:
This field is actually one level deeper in `author`
```suggestion
          author: {
            login: graphqlTypes.string,
          },
```

Reply:
Thank you.